### PR TITLE
AV-192470 : ALB-SDK update to 30.2.1 for AKO and AKO Gateway API 1.12

### PR DESCRIPTION
**Pull request by DixitAakash**

### CODE REVIEW

**Review by akshayhavile — APPROVED** [review]

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Comment by DixitAakash** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

File: internal/rest/avi_obj_httpps.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func (rest *RestOperations) AviHttpPSBuild(hps_meta *nodes.AviHttpPolicySetNode,
 		protocol := "HTTPS"
 		redirect_action.StatusCode = &hppmap.StatusCode
 		redirect_action.Protocol = &protocol
-		redirect_action.Port = &hppmap.RedirectPort
+		redirect_action.Port = uint32(hppmap.RedirectPort)
```
Comment (akshayhavile) [inline internal/rest/avi_obj_httpps.go:191]: @DixitAakash This field is changed from *int32 to uint32. Did we test this AKO image with older Avi Controller where these SDK changes are not there?

**Review by aaha97 — APPROVED** [review]
LGTM

File: internal/rest/avi_obj_httpps.go (hunk 1 above)
Comment (DixitAakash) [reply internal/rest/avi_obj_httpps.go:191]: @akshayhavile I have checked with controller version 22.1.3 and the changes are working fine without any issues.

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**